Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -909,7 +910,8 @@ static void webkitWebViewConstructed(GObject* object)
     if (!priv->websitePolicies)
         priv->websitePolicies = adoptGRef(webkit_website_policies_new());
 
-    webkitWebViewCreatePage(webView, webkitWebViewCreatePageConfiguration(webView));
+    Ref<API::PageConfiguration> configuration = priv->relatedView && priv->relatedView->priv->configurationForNextRelatedView ? priv->relatedView->priv->configurationForNextRelatedView.releaseNonNull() : webkitWebViewCreatePageConfiguration(webView);
```

Comment:
Can we just write `Ref` without the template argument?

Reply:
Probably...

Reply:
Works.